### PR TITLE
Project model should add a project tag

### PR DESCRIPTION
**Pull request by smo921**
The tag key name applied by a model's `defaults` should match the model class name.  It makes more sense that the Project model adds a `project` tag.

### CODE REVIEW

File: lib/kennel/models/project.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ module Models
     class Project < Base
       settings :team, :parts, :tags, :slack
       defaults(
-        tags: -> { ["project:#{kennel_id}"] + team.tags },
+        tags: -> { Array(["project:#{kennel_id}"] + team.tags).uniq },
```
Comment (smo921) [inline lib/kennel/models/project.rb]: Still need to do this in the subclasses, so this commit is probably unnecessary.

Comment (grosser) [reply lib/kennel/models/project.rb]: don't need the `Array`

**Comment by grosser** [comment]
needs a test-fix

yeah this is nice for consistency ... will suck a bit since a few links will break, but naming this service was a mistake :/

**Comment by smo921** [comment]
Completely forgot about this PR and not sure if it is necessary anymore.

**Comment by grosser** [comment]
so still need or not ?

**Comment by smo921** [comment]
I think so.  I hadn't rebased and looked at the code in a while. So I was
unsure if this had already been changed.

**Pull request closed without merging**